pandas/conftest.py

```diff
@@ -1432,3 +1432,17 @@ def __init__(self, **kwargs):
     registry.pop("testmem", None)
     TestMemoryFS.test[0] = None
     TestMemoryFS.store.clear()
+
+
+@pytest.fixture(
+    params=[
+        ("foo", None, None),
+        ("Egon", "Venkman", None),
+        ("NCC1701D", "NCC1701D", "NCC1701D"),
+    ]
+)
+def names(request):
```

Comment:
may want to make this fixture name more verbose (followon)

Reply:
followon yah, its used a bunch in arithmetic tests, and i actually need to do a pass for that anyway since we put some pretty thorough name-retention tests in tests.series.test_arithmetic semi-recently